yoga/YGEnums.h

```diff
@@ -21,6 +21,12 @@ typedef enum YGFlexDirection {
   YGFlexDirectionCount,
 } YGFlexDirection;
 
+typedef enum YGUnit {
+  YGUnitPixel,
+  YGUnitPercent,
+  YGUnitModeCount,
```

Comment:
YGUnitModeCount?? should be YGUnitCount probably. Did you forget to re-generate after changes?